res/styles.js

```diff
@@ -0,0 +1,28 @@
+import colors from "./colors"
```

Comment:
why it is here? remove it. All styles should be with components where they are used.